test/E2ETests/E2ETests/HttpEndToEndTests.cs

```diff
@@ -64,16 +64,6 @@ public async Task HttpTriggerTestsPocoResult()
             Assert.Equal($"{{{Environment.NewLine}  \"Name\": \"Test\"{Environment.NewLine}}}", responseBody);
         }
 
-        [Fact]
```

Comment:
We're actually in the process of bringing proxies back for v4. Is it possible to mark this one with a `Skip` for now, then we can get it running again once proxies are re-introduced?